shopinvader/models/shopinvader_variant.py

```diff
@@ -53,6 +53,23 @@ class ShopinvaderVariant(models.Model):
     price = fields.Serialized(
         compute='_compute_price',
         string='Shopinvader Price')
+    short_name = fields.Char(compute='_compute_name')
```

Comment:
@sebastienbeau IMO the plural form is more appropriate for this method. `_computes_names`